src/Knp/DictionaryBundle/Dictionary/StaticDictionaryFactory.php

```diff
@@ -2,9 +2,10 @@
 
 namespace Knp\DictionaryBundle\Dictionary;
 
+use Knp\DictionaryBundle\Dictionary;
 use Knp\DictionaryBundle\Dictionary\ValueTransformer\TransformerInterface;
 
-final class DictionaryFactory
+final class StaticDictionaryFactory
```

Comment:
I'm not sure about this "static dictionary dedicated factory".
As we talked earlier, I think we should keep the DictionaryFactory and introduce dedicated methods.